people_tool/views.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+from django.views.decorators.http import require_http_methods
+from lti_permissions.decorators import lti_permission_required
+from django_auth_lti import const
+from django_auth_lti.decorators import lti_role_required
+
+logger = logging.getLogger(__name__)
+
+
+# todo: re-enable lti_permission_required()
+@login_required
+@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
```

Comment:
Yes, since the migration are in there, we should  add the new setting for the entity 'people_tool' and modify it.